doc/source/scripts/gen_format_overview_classes.py

```diff
@@ -7,115 +7,167 @@
+            if "Coordinate writer" in handlers:
+                return SUCCESS
+            if "Converter" in handlers:
+                return SUCCESS
+            return FAIL
+
+        input_items = [
+            (format, handlers, _create_key(format, handlers))
+            for format, handlers in sorted_types
+        ]
+
+        self.table_writer = TableWriter(
+            filename="formats/format_overview.txt",
+            include_table="Table of all supported formats in MDAnalysis",
+            column_spec=[
+                ("File type", _file_type),
+                ("Description", _description),
+                ("Topology", _topology),
+                ("Coordinates", _coordinates),
+                ("Read", _read),
+                ("Write", _write),
+            ],
+            input_items=input_items,
+        )
+        self.table_writer.generate_lines_and_write_table()
+        self.table_writer.fields["keys"] = list(zip(*input_items))[2]
+
+
+class CoordinateReaders:
+    def __init__(self) -> None:
+        def _velocities(
+            fmt: str, handlers: dict[HANDLER_T, Type[Any]], key: str
+        ) -> str:
+            if handlers["Coordinate reader"].units.get("velocity", None):
+                return SUCCESS
+            return FAIL
+
+        def _forces(fmt: str, handlers: dict[HANDLER_T, Type[Any]], key: str) -> str:
+            if handlers["Coordinate reader"].units.get("force", None):
+                return SUCCESS
+            return FAIL
+
+        input_items = [
+            (format, handlers, _create_key(format, handlers))
+            for format, handlers in sorted_types
+            if "Coordinate reader" in handlers
+        ]
+        self.table_writer = TableWriter(
+            filename="formats/coordinate_readers.txt",
+            include_table="Table of supported coordinate readers and the information read",
+            input_items=input_items,
+            column_spec=[
+                ("File type", _file_type),
+                ("Description", _description),
+                ("Velocities", _velocities),
+                ("Forces", _forces),
+            ],
+        )
+        self.table_writer.generate_lines_and_write_table()
+
+
+class SphinxClasses:
+    def __init__(self, fmt: str):
+        def _custom_get_lines() -> list[list[str]]:
+            lines = []
+            for label, klass in sorted(FILE_TYPES[fmt].items()):
+                lines.append(
+                    [
+                        "**{}**".format(label),
+                        base.sphinx_class(klass=klass, tilde=False),
+                    ]
+                )
+            return lines
+
+        lines = _custom_get_lines()
+        self.table_writer = TableWriter(
+            filename="formats/reference/classes/{}.txt".format(fmt),
+            lines=lines,
+        )
+        self.table_writer.write_table()
+
+
+def main() -> None:
     ov = FormatOverview()
```

Comment:
```suggestion
    overview = FormatOverview()
```

Old me definitely went too much for brevity over interpretability!